docker/images/ubuntu1404/README.md

```diff
@@ -0,0 +1,38 @@
+# newman_ubuntu1404
+
+This image runs newman on node v8 on Ubuntu 14.04.2
+
+Build the image,
+
+```terminal
+docker build -t postman/newman_ubuntu1404 .
+```
+
+Or get it from [docker hub](https://registry.hub.docker.com/u/postman/newman_ubuntu1404/)
```

Comment:
please update the path on the Docker Hub
- https://hub.docker.com/r/postman/newman_ubuntu1404/